apps/prairielearn/src/lib/groups.ts

```diff
@@ -600,6 +600,17 @@ export async function updateGroupRoles(
   });
 }
 
+export async function deleteGroup(assessment_id: string, group_id: string, authn_user_id: string) {
+  const deleted_group_id = await sqldb.queryOptionalRow(
+    sql.delete_group,
+    { assessment_id, group_id, authn_user_id },
+    IdSchema,
+  );
+  if (deleted_group_id == null) {
+    throw new Error('Group does not exist.');
```

Comment:
403/404 instead?

Reply:
Done in 74af8894a6b0571eac1c04f2d95ef0e31fe4437b.